lib/sda/host_test.go

```diff
@@ -250,7 +250,7 @@ func TestTreePropagation(t *testing.T) {
 		network.DumpTypes()
 		t.Fatal("h1 didn't receive SendTree type:", msg.MsgType)
 	}
-	if msg.Msg.(sda.TreeMarshal).EntityId != uuid.Nil {
+	if msg.Msg.(sda.TreeMarshal).EntityListID != sda.EntityListID(uuid.Nil) {
```

Comment:
I added an `Equals()` method, if we were comparing them in production code. But I didn't focus on trying to make the tests "less noisy". I would prefer to merge it and open another issue: #324